Doc/Tutorial/chapter_seq_objects.tex

```diff
@@ -13,14 +13,14 @@ \chapter{Sequence objects}
 
 \section{Sequences and Alphabets}
 
-The alphabet object is perhaps the important thing that makes the \verb|Seq| object more than just a string. The currently available alphabets for Biopython are defined in the \verb|Bio.Alphabet| module. We'll use the \href{http://www.sbcs.qmul.ac.uk/iupac/}{IUPAC alphabets} here to deal with some of our favorite objects: DNA, RNA and Proteins.
+Historically the \verb|Seq| object in Biopython ``.alphabet`` attribute was a
+complex object which captured not just the type of sequence (like DNA, RNA, or
+Protein) but additional information like the expected letters or gap character.
+This has been cut in Biopython 1.78.
```

Comment:
In other places the documentation refers to historical changes in Biopython (going back to release 1.45 from twelve years ago). I think it doesn't belong in the documentation. It may appear important now, but it quickly makes our documentation look out of date.

Reply:
Have you worked with much legacy code? You might be surprised. Personally I don't find this makes the documentation look out of date, Python's own documentation has plenty of notes about when something was added.

I think we need something here for about the removal of the old alphabet stuff, at least for a year.

I and others have periodically done sweeps of the code for removing obsolete references - it may be timely for another sweep.

Reply:
I don't think the FAQ section is a good place. Currently the FAQ describes ancient changes from 10 years ago which, in 2020, are not frequently-asked questions. FAQ should cover questions that are actually frequently asked.
We could have a separate chapter in the documentation to collect this information on the historical developments in Biopython.

Reply:
That makes sense - the proposed chapter can include any key examples, and can cross reference the more terse DEPRECATED file.